src/index.js

```diff
@@ -122,6 +122,9 @@ async function run (cwd, config) {
     const eslint = require(eslintPath)
     engine = new eslint.CLIEngine({
       parser: babelEslintPath,
+      parserOptions: {
+        requireConfigFile: false
+      },
```

Comment:
This solves the babel parsing error that throws for each file after upgrading to `@babel/eslint-parser`:

```
./prettier-standard/src/cli.js
  0:0  error  Parsing error: No Babel config file detected for ./prettier-standard/src/cli.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files

./prettier-standard/src/index.js
  0:0  error  Parsing error: No Babel config file detected for ./prettier-standard/src/index.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files

./prettier-standard/src/scms/git.js
  0:0  error  Parsing error: No Babel config file detected for ./prettier-standard/src/scms/git.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files

./prettier-standard/src/utils.js
  0:0  error  Parsing error: No Babel config file detected for ./prettier-standard/src/utils.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files

./prettier-standard/test/cli.test.js
  0:0  error  Parsing error: No Babel config file detected for ./prettier-standard/test/cli.test.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files

./prettier-standard/test/index.test.js
  0:0  error  Parsing error: No Babel config file detected for ./prettier-standard/test/index.test.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files
```